ci-operator/step-registry/hypershift/mce/aws/create/hypershift-mce-aws-create-chain.yaml

```diff
@@ -19,6 +19,9 @@ chain:
         - name: ENABLE_ICSP
           default: "false"
           documentation: "if true, add image content sources config(path=${SHARED_DIR}/mgmt_icsp.yaml)"
+        - name: HYPERSHIFT_INSTANCE_TYPE
+          default: "m5.xlarge"
+          documentation: "Instance type for the cluster nodes"
         - name: BREW_IMAGE_REGISTRY_USERNAME
           default: '|7820aea2-0d75-11e7-9259-28d244ea5a6d.hhav.f63e13'
```

Comment:
Not related to this PR. But I think this kind of info should be stored in vault. CC @YuLi517  @jianlinliu @jhou1

Reply:
agree.

Reply:
Since these steps are maintained by multiple teams and I am not the owner of BREW_IMAGE_REGISTRY_USERNAME, I need to communicate with the other teams and then open a PR to take it.

Reply:
I'm fine to let this PR go.  However it's better to make sure there are jira issues to track the mentioned issues, especially this one.

Reply:
create https://issues.redhat.com/browse/HOSTEDCP-1949 to check